### ISSUE TITLE
Follow up for "scheduled sending" and "undo sending" features 

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/mail#6242]
### Is your feature request related to a problem? Please describe.

Related to: #5884 

### Todo (ordered by importance)

- [x] #6246
- [x] #6268
- [x] #6266
- [x] #6269
----
- [x] #6248
- [x] #6292
- [x] #6247
- [x] #6267
- [x] #6243
- [x] #6244
- [x] #6288 

## Critical bugs and other follow-ups
* [x] https://github.com/nextcloud/mail/issues/6157
* [x] https://github.com/nextcloud/mail/issues/6161
* [x] https://github.com/nextcloud/mail/issues/6189
* [ ] https://github.com/nextcloud/mail/issues/6194
* [x] https://github.com/nextcloud/mail/issues/6195
* [ ] #6156

**Comment by miaulalala** [comment]
- [ ] Scrollbar is overflowing composer modal
- [ ] Rename "Outbox Draft" into "Edit Message" for composer modal title
- [ ] Move Deletion Modal into Toast
- [ ] Outbox envelopes on hover jump around because of missing subject line
- [ ] Add Outbox Description in rightmost column as content is empty otherwise
- [ ] Set default date and time for "custom send time" to one hour into the future

**Comment by ChristophWurst** [comment]
@miaulalala @JuliaKirschenheuter could you please convert the tasks of this ticket into standalone tickets? Think: *could someone else understand the problem of the description without following all linked parent tickets?* Cases like https://github.com/nextcloud/mail/issues/6273 will be hard to follow in a few weeks time when we don't have our minds occupied with the outbox feature anymore. Try to make this a decent bug report or feature request ticket, whatever is more appropriate. The four important bits of this tickets are done and therefore I would close this one.

Cheers

**Comment by ChristophWurst** [comment]
@miaulalala @JuliaKirschenheuter could one of you please take care?

**Comment by miaulalala** [comment]
@ChristophWurst closing this now as all tickets are able to be understood on their own.
